CICE-interface/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ list(APPEND lib_src_files
   ${icepack_files}
   ${cice_mpi_comm_files}
   ${cice_nuopc_cmeps_driver_files}
-  ${cice_cdeps_inline_files})
+  ${cice_cdeps_share_files})
```

Comment:
I know this is a temporary solution. Does this mean that now cice/mom/ww3 caps now have dependency on the CDEPS share code?

Reply:
Yes. 

CICE and CMEPS already have dependency for CDEPS inline functionality.

MOM6 and WW3 now depend on restart_fh module. While this is currently under CDEPS-interface, it is a ufs-weather-model file (not a CDEPS subcomponent file)